src/NewsService.Mappers/Models/NewsInfoMapper.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using LT.DigitalOffice.Kernel.Broker;
+using LT.DigitalOffice.Models.Broker.Requests.Company;
+using LT.DigitalOffice.Models.Broker.Requests.User;
+using LT.DigitalOffice.Models.Broker.Responses.Company;
+using LT.DigitalOffice.Models.Broker.Responses.User;
+using LT.DigitalOffice.NewsService.Mappers.Models.Interfaces;
+using LT.DigitalOffice.NewsService.Models.Db;
+using LT.DigitalOffice.NewsService.Models.Dto.Models;
+using MassTransit;
+using Microsoft.Extensions.Logging;
+
+namespace LT.DigitalOffice.NewsService.Mappers.Models
+{
+  public class NewsInfoMapper : INewsInfoMapper
+  {
+    private IRequestClient<IGetUserDataRequest> _userRequestClient;
```

Comment:
better use getUsersData, we don't want to use requests for one entity

Reply:
rework it in other task

Reply:
i just use edditorconfig

Reply:
readonly